site/content/docs/commands/secret-init.en.md

```diff
@@ -8,9 +8,6 @@ $ copilot secret init
 
 A secret can have different values in each of your existing environments, and is accessible by your services or jobs from the same application and environment.
 
-!!! attention 
-    Secrets are not supported for Request-Driven Web Services.
-
```

Comment:
this change seems to happen before. did you work on the latest commit in mainline?

Reply:
This was already pushed and merged. I committed on the same branch again. Should I send out a new PR?